packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCard.tsx

```diff
@@ -23,8 +24,14 @@ export class DocumentCard extends React.Component<IDocumentCardProps, any> {
       };
     }
 
+    // if this element is actionable it should have an aria role
+    let role = actionable ? (onClick ? 'button' : 'link') : null;
+    let tabIndex = actionable ? 0 : null;
+
     return (
       <div
+        tabIndex={ tabIndex }
```

Comment:
we shouldn't assign tabIndex to it. For example, we use DocumentCard in activity webpart, we don't want every documentCard to be tab stopper, we rely on FocusZone to make only one document card is tab stopper, and use left/right/up/down key to navigate to other documentCard.